test/src/code_field/split_highlight_lines_test.dart

```diff
@@ -38,58 +15,73 @@ public class MyClass {
   }
 }''',
     expected: '''
-{value: 'public '}
-{className: class, children: ({value: ''}, {className: keyword, children: ({value: 'class'})}, {value: ' '}, {className: title, children: ({value: 'MyClass'})}, {value: ' '})}
-{value: '{
-'}
-{value: '  public '}
-{className: keyword, children: ({value: 'void'})}
-{value: ' main() { '}
-{className: comment, children: ({value: '// readonly'})}
-{value: '
-'}
-{value: '  }
-'}
-{value: '}'}''',
+{value: 'public ', children: empty}
+{className: class}
+  {value: '', children: empty}
+  {className: keyword}
+    {value: 'class', children: empty}
+  {value: ' ', children: empty}
+  {className: title}
+    {value: 'MyClass', children: empty}
+  {value: ' ', children: empty}
+{value: '{\n', children: empty}
+{value: '  public ', children: empty}
+{className: keyword}
+  {value: 'void', children: empty}
+{value: ' main() { ', children: empty}
+{className: comment}
+  {value: '// readonly', children: empty}
```

Comment:
'readonly' is non-neutral, it grabs attention and suggests what it is not. Use something like 'comment' for meaningless comments.